react-native-contacts.podspec

```diff
@@ -13,4 +13,6 @@ Pod::Spec.new do |s|
   s.source         = { :git => package_json["repository"]["url"] }
   s.source_files   = 'ios/RCTContacts/*.{h,m}'
 
+  s.dependency 'React'
```

Comment:
this was removed a while back on https://github.com/rt2zz/react-native-contacts/commit/efac356a1bf7dbbc248e092300f44475db137e92#diff-26fb79079f7ac459de7c6d05775f7955
is there another way to solve the problem?

This pod is listed as deprecated on https://cocoapods.org/pods/React

Reply:
When using `cocoapods `to install RN dependencies like `react-native-contacts` and so on, developer should *also* install `React` using `cocoapods`. When a dependency like `React` point to local folders has been defined in the `Podfile` of the project, the `React` dependency defined in the `react-native-contacts.podspec` is point to the same `React`.

Developer install `React` of 0.55.4 should like:
```

node_modules_path = "../node_modules"
# some other dependencies like react-native-contacts
pod "react-native-contacts", :path => "#{node_modules_path}/react-native-contacts"

# react-native dependencies
react_native_path = "#{node_modules_path}/react-native"

pod "yoga", :path => "#{react_native_path}/ReactCommon/Yoga"
  pod "DoubleConversion", :podspec => "#{react_native_path}/third-party-podspecs/DoubleConversion.podspec"
  pod "glog", :podspec => "#{react_native_path}/third-party-podspecs/glog.podspec"
  pod "Folly", :podspec => "#{react_native_path}/third-party-podspecs/Folly.podspec"

# React
pod "React", :path => react_native_path, :subspecs => [
      "Core",
      #"BatchedBridge",
      "CxxBridge",
      "DevSupport",
      #"tvOS",
      #"jschelpers_legacy",
      #"cxxreact_legacy",
      "jschelpers",
      "PrivateDatabase",
      "cxxreact",
      "ART",
      "RCTActionSheet",
    #"RCTAdSupport",
      "RCTAnimation",
      "RCTBlob",
      "RCTCameraRoll",
      "RCTGeolocation",
      "RCTImage",
      "RCTNetwork",
      "RCTPushNotification",
      "RCTSettings",
      "RCTText",
      "RCTVibration",
      "RCTWebSocket",
      "fishhook",
      "RCTLinkingIOS",
      "RCTTest",
    #"_ignore_me_subspec_for_linting_",
      ]
```

Reply:
so you are saying that we should list `React` in the podfile because it will point to the local copy of react, not the deprecated cocoapod. Is this behavior documented somewhere?

Reply:
Sure, because in `Podfile` we add `React` dependency from local path using `:path => react_native_path`. When `pod install`, `cocoapods` will find files from this path instead of from a remote repository.
I read this somewhere and cannot find it from [cocoapods.org](https://cocoapods.org/) now, so you can just try to confirm.
Remember to *remove* `React` from current project first and then add code abow to install `React`.